### PR TITLE
Add static tweets, subdomain auth, and domains API examples.

### PR DESCRIPTION
**Pull request by steven-tey**

### CODE REVIEW

File: domains-api/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+**One-Click Deploy**
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=platforms-eap):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel-customer-feedback/platforms/tree/main/code-snippets/domains-api&project-name=domains-api&repository-name=domains-api)
```
Comment (leerob) [inline domains-api/README.md:68]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/domains-api&project-name=domains-api&repository-name=domains-api)
```

File: domains-api/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+**Clone and Deploy**
+
+Download this repository via git:
+
+```bash
+git clone https://github.com/vercel-customer-feedback/platforms.git
```
Comment (leerob) [inline domains-api/README.md:75]:
```suggestion
git clone https://github.com/vercel/examples/tree/main/domains-api.git
```

Will this work?

File: domains-api/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npx create-next-app --example platforms/code-snippets/domains-api domains-api
+# or
+yarn create next-app --example platforms/code-snippets/domains-api domains-api
```
Comment (leerob) [inline domains-api/README.md:83]:
```suggestion
npx create-next-app --example https://github.com/vercel/examples/tree/main/domains-api domains-api
# or
yarn create next-app --example https://github.com/vercel/examples/tree/main/domains-api domains-api
```

Does this work?

File: domains-api/pages/api/hello.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default function helloAPI(req, res) {
```
Comment (leerob) [inline domains-api/pages/api/hello.js:3]: Can be removed

File: domains-api/pages/api/remove-domain.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+export default async function removeDomain(req, res) {
+
+    const { domain } = req.query;
+    
+    const response = await fetch(`https://api.vercel.com/v8/projects/${process.env.VERCEL_PROJECT_ID}/domains/${domain}?teamId=${process.env.VERCEL_TEAM_ID}`, {
```
Comment (leerob) [inline domains-api/pages/api/remove-domain.js:5]: Run prettier on files? Can be post merge

File: domains-api/pages/index.js [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+import useSWR, { mutate } from "swr";
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+import Image from "next/image";
+
+export default function Home() {
+  const [domain, setDomain] = useState("");
+  const [domainList, setDomainList] = useState(
+    Cookies.get("domainList")
+      ? JSON.parse(Cookies.get("domainList"))
+      : ["platformizer.co", "vercel.pub"]
```
Comment (leerob) [inline domains-api/pages/index.js:15]: Is this needed?

File: domains-api/pages/index.js [hunk 7]
```diff
@@ -0,0 +1,348 @@
...
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Toaster
+        position="bottom-right"
+        toastOptions={{
+          duration: 10000,
+        }}
+      />
+      <a
+        href="https://github.com/vercel-customer-feedback/platforms/tree/main/code-snippets/domains-api"
```
Comment (leerob) [inline domains-api/pages/index.js:50]: Just do a find all for this and replace 😄

File: domains-api/tailwind.config.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  mode: 'jit',
```
Comment (leerob) [inline domains-api/tailwind.config.js:2]: Post merge: update deps

File: static-tweets-tailwind/README.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+**One-Click Deploy**
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=platforms-eap):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel-customer-feedback/platforms/tree/main/code-snippets/static-tweets-tailwind&project-name=static-tweets-tailwind&repository-name=static-tweets-tailwind)
```
Comment (leerob) [inline static-tweets-tailwind/README.md:30]: Same comments as above for updating URLs here

File: static-tweets-tailwind/pages/api/hello.js [hunk 10]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default function helloAPI(req, res) {
```
Comment (leerob) [inline static-tweets-tailwind/pages/api/hello.js:3]: Delete

File: subdomain-auth/README.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+**Clone and Deploy**
+
+Download this repository via git:
+
+```bash
+git clone https://github.com/vercel-customer-feedback/platforms.git
+```
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
```
Comment (leerob) [inline subdomain-auth/README.md:28]: Same 😄

File: subdomain-auth/package.json [hunk 12]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "simple-example",
+  "version": "1.0.0",
+  "repository": "https://github.com/vercel/examples.git",
+  "license": "MIT",
```
Comment (leerob) [inline subdomain-auth/package.json:5]:
```suggestion
  "private": true,
```

**Comment by leerob** [comment]
CleanShot 2022-01-20 at 11 23 37@2x
Make sure this is updated 👍

**Review by SuttonJack — APPROVED** [review]

**Review by leerob — APPROVED** [review]

**Pull request merged**